pkgs/development/python-modules/pygal/default.nix

```diff
@@ -29,11 +29,11 @@ buildPythonPackage rec {
 
     # Should be a check input, but upstream lists it under "setup_requires".
     # https://github.com/Kozea/pygal/issues/430
-    pytestrunner
+    (pytestrunner.overrideAttrs (oldAttrs: oldAttrs // { pytest = pytest_3; }))
```

Comment:
pytestrunner.override

Reply:
Yeah I should probably close this PR if I'm not going to update it and focus on the `-staging` rebase instead https://github.com/NixOS/nixpkgs/pull/60888